spec/rails_helper.rb

```diff
@@ -67,7 +67,7 @@
   # Prevent unintended API access from Geocoder
   config.before :each do
     allow(Geocoder).to receive(:search).and_return([
-      OpenStruct.new(latitude: 53.4794892, longitude: -2.2451148)
+      Geocoder::Result::Test.new(latitude: 53.4794892, longitude: -2.2451148)
```

Comment:
Should this change be in here - I thought this related to a different change?

Reply:
I switched the specs over from `OpenStruct` to `Geocoder::Result::Test` in #141 but now we're being a bit more careful about what goes into and comes out of the cache it needed to be done here too. The 'sorting by distance' feature will now fail without this change.